### ISSUE TITLE
Need to incorporate different heights of dots

### ISSUE THREAD

**Issue by therzberg** [daisy/ebraille#55]
Multiple heights of dots are necessary for high-quality tactile graphics.

Detail
There are some tactile graphics (such as two parallel lines cut by a transversal) that do not require multiple heights of dots. However, when graphing in the coordinate plane, you need grid lines at the lowest level of dots, the x- and y-axes at the 2nd level of dots, and the graph itself at the third level of dots at a minimum. Some graphics on the coordinate plane require textures in addition to the previously mentioned items. 

Mathematics is pretty simple compared to the social studies maps and scientific diagrams that are often included in textbooks and high-stakes assessments. These graphics require not only a different height of dots, but they also involve many different textures that must somehow be dealt with in just the right combination so as not to become cluttered nor become too small and/or too similar to distinguish.

Proposal
The eBRF should be designed to accommodate multiple heights of dots and include meta-data that allows end users to customize the graphic for their specific hardware.

Submitted by Susan Osterhaus and Tina Herzberg

**Comment by jrbowden** [comment]
I agree multi-height graphics can be useful, but not all braille hardware will be able to display different height dots, so there needs to be a graceful fallback position.

Metadata is needed to indicate that multi-height is requirement for a particular document.

Out of interest, what about text? If we already know (by the mark-up, see #17), it could theoretically be possible to render bold text at a different dot height as an option.

**Comment by andrewflatres1** [comment]
I love the idea of that for the bold representation. I guess this should be more of a desire than an MVP.



Thanks
Andrew

**Comment by wfreeman-aph** [comment]
I think it's possible that we could leave dealing with multi-height graphics (and potentially braille) to the reading system. If a graphic requires multiple heights and the hardware cannot reproduce it, then it should be on the hardware to decide how to handle that by either finding a suitable replacement or by letting the user know that the graphic cannot be accurately reproduced using the current hardware. All we have to do is provide metadata that makes it clear that a specific graphic has multiple heights and file-level metadata that indicates the file contains multi-height graphics.

On multi-height braille, I would say that's somewhat dangerous territory but, again, something that could be left to the reading system. All our file type needs to indicate is that the text is bold, if the reading system wants to make that clear to the user by using height, then that's a decision they can make when rendering the document. That's what I'm thinking anyway.
